packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -129,3 +134,5 @@ export function validateOrigin(messageOrigin: URL): Promise<boolean> {
     return false;
   });
 }
+
+prefetchOriginsFromCDN();
```

Comment:
Can you add new unit tests to `validOrigins.spec.ts` that validate the fetch happens when the library is loaded?

Reply:
I'm unsure how I would do this since now every unit test actually runs through the fetch code. This is why I needed to add the setupTests.ts file to mock fetch earlier in the jest pipeline otherwise it would complain about `fetch` not existing.